examples/random_agent.py

```diff
@@ -4,16 +4,18 @@
 import retro
 
 parser = argparse.ArgumentParser()
-parser.add_argument('game', help='the name or path for the game to run')
-parser.add_argument('state', nargs='?', help='the initial state file to load, minus the extension')
```

Comment:
Why these changes? Definitely should be a separate commit.

Reply:
Removed